app/controllers/api/meetings_controller.rb

```diff
@@ -9,6 +10,7 @@ def index
       render json: @meetings
     else
       meetings = Meeting.all
+      meetings = meetings.where(participatory_process_id: @participatory_process.try(:id))
```

Comment:
Same as up above. Also, I'm sure we can somehow remove all this boilerplate. Maybe providing a `scoped_resource` method in the `ParticipatoryProcess` concern that will do that for you?